.github/workflows/main.yml

```diff
@@ -40,3 +40,7 @@ jobs:
       - run: sudo make -j4 package
         shell: bash
         name: Build
+      - uses: actions/upload-artifact@v1
+        with:
+          name: ${{ format( 'packages-{0}.tgz', matrix.os) }}
```

Comment:
Can you explain this line?

What is `packages-`?   

Is there any way to test this?    Don't we want to limit this to only uploading for tags?

Reply:
I renamed it `dist` to be more consistent, and added a comment. 

The upload appears in the "Actions" tab on GH, and its not actually a release. It just allows us to download the Actions-built contents of `dist`, untar that into the actual built artifacts, and then upload those to a tag. I gave up on programming that sequence in yml, so we can do that by hand until someone dles.

Reply:
Sweet!  So we can get a copy of the sdk for each pull request.  Nice.  Out of interest do you know are disk space limits enforced on this kind of resource?